FossClubWebsite/WebContent/HomePage.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html ng-app="website">
+<head>
+<meta charset="UTF-8">
+<title>HomePage.html</title>
+<link rel="stylesheet" href="css/bootstrap/bootstrap.css"/>
+<link rel="stylesheet" href="css/customstyle.css"/>
+<script src="js/jquery/jquery-3.2.1.js"></script>
+<script src="js/bootstrap/bootstrap.js"></script>
+<script src="js/app.js"></script>
+</head>
+<body>
+ <!-- Navigation Bar Start -->
+ <nav class="navbar navbar-inverse navbar-fixed-top">
+ <div class="container" w3-padding = "16">
+ <div class="navbar-header">
+ <a class="navbar-brand" href="HomePage.html">Foss Club</a>
+ </div>
+ <div id="navbar" class="navbar-collapse collapse">
+ <form class="navbar-form navbar-right">
+ <a class="navbar-brand" href="HomePage.html">Contributors</a>
```

Comment:
@Sush97 @Pranavya try to make templates for frequently used UI components on a page. For example, if the nav bar is repeated in every tab, then make it as a single HTML file and simply include in the header. 

The same way for other partial pages like Projects, Hall of Fame...so on.